src/app/ssmt/ssmt-report/losses-summary/losses-summary.component.ts

```diff
@@ -79,25 +79,25 @@ export class LossesSummaryComponent implements OnInit {
   }
 
   checkCondensingTurbine(loss: SSMTLosses) {
-    if (loss.condensingTurbineEfficiencyLoss || loss.condensingTurbineUsefulEnergy) {
+    if (loss.condensingTurbineEfficiencyLoss >= 0|| loss.condensingTurbineUsefulEnergy) {
```

Comment:
Here you would check the new booleans added

Reply:
These changes have now been pushed